lib/cassandra.js

```diff
@@ -211,7 +211,7 @@ Cassandra.prototype.executeSQL = function(cql, params, options, callback) {
    return callback && callback(err, data ? data.rows : null);
   }
 
-  this.client.execute(cql, serialize(params), { prepare : true, readTimeout: 30000 }, myCallback);
+  this.client.execute(cql, serialize(params), { prepare : true, readTimeout: this.settings.readTimeout }, myCallback);
```

Comment:
If this value isn't set, will this work, or will it explode? There should be a default value here.
(`readTimeout: this.settings.readTimeout || 30000`)

Reply:
change is ready, in new pull request

Reply:
Sorry, are you saying that this PR is obsolete or that the other one is a complement to this one?